contentcuration/contentcuration/static/js/edit_channel/exercise_creation/views.js

```diff
@@ -715,7 +719,6 @@ var ExerciseEditableListView = BaseViews.BaseEditableListView.extend({
         }
     },
     propagate_changes:function(){
-        this.validate();
```

Comment:
Not actually as important here- it was causing the hint modal to display an error as soon as a hint was added, so this is a little less angry now